crates/sovereign-sdk/full-node/db/sov-schema-db/src/iterator.rs

```diff
@@ -120,6 +123,15 @@ where
             ScanDirection::Backward => self.db_iter.prev(),
         }
 
+        let v = Instant::now().saturating_duration_since(start);
+        SCHEMADB_ITER_LATENCY_SECONDS
+            .histogram
+            .get_or_create(&vec![(
+                "cf_name".to_owned(),
+                S::COLUMN_FAMILY_NAME.to_owned(),
+            )])
```

Comment:
get_or_create is used everywhere in sov-schema-db, but we already know the dbs, and column families. can we somehow create these at the startup as well?